redash/settings/helpers.py

```diff
@@ -39,7 +39,7 @@ def set_from_string(s):
 
 
 def parse_boolean(str):
-    return json.loads(str.lower())
+    return simplejson.loads(str.lower())
```

Comment:
It's not related to this PR, but isn't it simpler to just compare value to `'true'`/`'false'` string instead of using JSON parser?

Reply:
yeah, that's easier and slightly faster, too